src/main/java/quack/Ui.java

```diff
@@ -0,0 +1,252 @@
+package quack;
+
+import java.util.Scanner;
+
+import quack.exception.InvalidIndexException;
+import quack.exception.InvalidTaskTypeException;
+
+import quack.tasks.Task;
+
+/**
+ * This class is responsible for displaying any UI interface,
+ * for the Quack chatbot.
+ */
+public class Ui {
+    
+    /** String to print out the spacers between each command */
+    private String spacer = "-".repeat(65);
```

Comment:
Add a constant to avoid magic numbers (65 in this case)?

Reply:
Thanks for spotting this out. I have removed this line of code after implementing JavaFX